flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+      type: object
+      properties:
+        version:
+          x-field-pattern:
+            description: >-
+              Protocol Version number. 
+            format: integer
+            length: 4
+            default: 1
+            features: [count]
+          x-field-uid: 1
+        flag:
+          description: >-
+              Flag, 0x01-0x08: Reserved.
+          $ref: '#/components/schemas/Flow.RSVP.Flag'
+          x-field-uid: 2
+        rsvp_checksum:
+          x-field-pattern:
+            description: >-
+              The one's complement of the one's complement sum of the message, with the checksum field replaced by zero for the purpose of computing the checksum.  
+              An all-zero value means that no checksum was transmitted.
+            format: checksum
+            length: 16
+          x-field-uid: 3
+        time_to_live:
+          x-field-pattern:
+            description: >-
+              The IP time-to-live(TTL) value with which the message was sent.
+            format: integer
+            length: 8
+            default: 0
+            features: [count]
+          x-field-uid: 4
+        reserved:
+          x-field-pattern:
+            description: >-
+              Reserved 
+            format: integer
+            length: 8
+            default: 0
+            features: [count]
+          x-field-uid: 5
+        rsvp_length:
+          description: >-
+            The sum of the lengths of the common header and all objects included in the message.
+          $ref: '#/components/schemas/Flow.RSVP.Length'
+          x-field-uid: 6
+        message_type:
+          description: >-
+            An 8-bit number that identifies the function of the RSVP message.
+            There are aound 20 message types is defined https://www.iana.org/assignments/rsvp-parameters/rsvp-parameters.xhtml#rsvp-parameters-2 .
+            Among Those we are presently supporting "path"(value: 1) message type.
+          $ref: '#/components/schemas/Flow.RSVP.Message'
+          x-field-uid: 7
+    Flow.RSVP.Flag:
+      type: object
+      properties:
+        choice:
+            type: string
+            default: not_refresh_reduction_capable
+            x-enum:
+              not_refresh_reduction_capable:
+                x-field-uid: 1
+              refresh_reduction_capable:
+                x-field-uid: 2
+            x-field-uid: 1
+    Flow.RSVP.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
+            If the OTG is unable to generate a value the default value must be used.
```

Comment:
The OTG implementation can provide a system generated value for this property.
            If the OTG is unable to generate a value the default value must be used.-> OTG will provide a system generated value for this property.  If OTG is unable to generate a value the default value must be used

Reply:
Done.